build-java/action.yml

```diff
@@ -119,7 +119,7 @@ runs:
         GITHUB_PACKAGES_USERNAME: ${{ inputs.tokenUsername }}
         GITHUB_PACKAGES_TOKEN: ${{ inputs.token }}
       shell: bash
-      run: mvn -s settings.xml sonar:sonar -Dsonar.host.url=${{ inputs.sonarUrl }} -Dsonar.login=${{ inputs.sonarToken }}
+      run: mvn -s settings.xml -DargLine="--add-opens=java.base/java.lang=ALL-UNNAMED" sonar:sonar -Dsonar.host.url=${{ inputs.sonarUrl }} -Dsonar.login=${{ inputs.sonarToken }}
```

Comment:
Not sure if required here as only required for running the tests. Being on the safe side. Will probably move away from Orika mapper long term so this should only be required in the interim.